src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Services/SyntaxFacts/CSharpSyntaxFacts.cs

```diff
@@ -620,9 +617,6 @@ public bool IsInConstructor(SyntaxNode node)
     public bool IsUnsafeContext(SyntaxNode node)
         => node.IsUnsafeContext();
 
-    public SyntaxNode GetNameOfAttribute(SyntaxNode node)
-        => ((AttributeSyntax)node).Name;
```

Comment:
moved to common ISyntaxFacts pattern where there is `GetPartsOfXXX` (GetPartsOfAttribute in this case), and the `GetXxxOfYYY` is now an extension that defers to that.